### ISSUE TITLE
Question : Parse a string to create Equation ?

### ISSUE THREAD

**Issue by nareshjois** [dharmatech/Symbolism#4]
Is there already a way in which I can create an equation using a string rather than static typing it.

**Comment by Happypig375** [comment]
Bump. A symbolic math library without a parser? Absolute blocker for me. The search for a symbolic math library with a parser continues.

**Comment by sharpninja** [comment]
Yup, when I first read about this library on reddit I immediately thought it would parse strings.  It would not be difficult to create a grammar with antlr4 and use the object model in the evaluation tree.
